_data/engine-cli-edge/docker_attach.yaml

```diff
@@ -64,72 +13,4 @@ options:
 - option: sig-proxy
   default_value: "true"
   description: Proxy all received signals to the process
-examples: |-
```

Comment:
I don't know why the same thing would have been repeated in examples but it was wrong anyway.